pkg/apis/servicecatalog/validation/broker_test.go

```diff
@@ -208,12 +229,56 @@ func TestValidateServiceBroker(t *testing.T) {
 					Name: "test-broker",
 				},
 				Spec: servicecatalog.ServiceBrokerSpec{
-					URL:      "http://example.com",
-					CABundle: []byte("fake CABundle"),
+					URL:            "http://example.com",
+					CABundle:       []byte("fake CABundle"),
+					RelistBehavior: "Duration",
+					RelistDuration: &metav1.Duration{15 & time.Minute},
+				},
+			},
+			valid: true,
+		},
+		{
+			name: "valid broker - manual behavior with behavior",
```

Comment:
nit: name should be manual behavior with **duration**   instead of manual behavior with beavior